vmm/src/cpu.rs

```diff
@@ -811,6 +811,8 @@ impl CpuManager {
     ) -> Result<()> {
         let reset_evt = self.reset_evt.try_clone().unwrap();
         let exit_evt = self.exit_evt.try_clone().unwrap();
+        #[cfg(feature = "kvm")]
```

Comment:
Please fix both your commit title and your commit message, replacing `exists` with `exits`.

Reply:
@russell-islam You still missed a typo in the commit message body.
> MSHV does not require to ensure MMIO/PIO exists complete
before pausing. This patch makes sure the above requirement
by checking the hypervisor type run-time.